src/main/java/org/apache/commons/imaging/formats/tiff/datareaders/DataReaderStrips.java

```diff
@@ -14,6 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+ /*
+ * Implementation notes:
+ *    See ImageDataReader for notes on development
+ * with particular emphasis on run-time performance.
+ */
```

Comment:
I think this comment could go above the class declaration, and maybe use the @link or @see, etc Javadocs annotations?